src/web/app/components/question-types/question-edit-answer-form/mcq-question-edit-answer-form.component.ts

```diff
@@ -81,4 +92,17 @@ export class McqQuestionEditAnswerFormComponent
     });
   }
 
+  /**
+   *
+   */
+  updateSelectedMcqDropdownOption($event: Event): void {
+    // @ts-ignore
```

Comment:
Why is `@ts-ignore` needed here? Perhaps it will be good to add a comment explaining it.